StripeIdentity/StripeIdentity/Source/NativeComponents/Views/IdentityHTMLView/HTMLTextView.swift

```diff
@@ -66,13 +66,53 @@ final class HTMLTextView: UIView {
         fatalError("init(coder:) has not been implemented")
     }
 
+    // Check if htmlText is <ul></ul>, if so, return the <li> within, return nil otherwise
+    fileprivate func tryParseUl(htmlString: String) -> [String]? {
```

Comment:
is it possible to add tests for this?

Reply:
thanks for catching this, this indeed breaks something - it matches partially within the html text, so if there's something before or after `<ul></ul>` they'll all be ignored. 
Just updated the logic and corresponding test